src/__tests__/DashbaordNav.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { BrowserRouter as Router} from 'react-router-dom';
+import {fireEvent, render} from '@testing-library/react';
+import DashboardNav from '../components/DashboardNav/DashboardNav'
+
+
+
+describe('<DashboardNav />', () => {
+    it('should render the DashboardNav component', ( )=> {
+        render(
+            <Router>
+                <DashboardNav />
+            </Router>
+        )
+        expect(<DashboardNav />).not.toBeNull()
+    })
+
+    it('elements', () => {
+         <Router>
+             <DashboardNav/>
+         </Router>  
+    })
+})
+
```

Comment:
Kindly revisit **DashboardNav.js** tests.. it seems you tested the existence of document in the DOM but some functionalities are not yet covered.

Reply:
Hi! kindly review again! now test are raised